default/stringtables/en.txt

```diff
@@ -7935,57 +7943,113 @@ Launch Bay
 
 PC_FIGHTER_BAY_DESC
 '''Launch bays are the access point for [[encyclopedia FIGHTER_TECHS]] to enter or exit a ship.
-Each launch bay allows a number of fighters to launch each combat round.
-'''
+Launched fighters join combat the round after being launched, returning once combat ends for that turn.'''
 
 # Ship part classes
 PC_SHIELD
 Shields
 
+PC_SHIELD_DESC
+'''Shield parts reduce the [[encyclopedia DAMAGE_TITLE]] sustained from each weapon hit by the shield parts strength.
+
+Unlike a planet shield, a ship shield is a reflective shield and does not sustain any loss in strength from an attack. Ship shields are ineffective against [[encyclopedia FIGHTER_TECHS]].'''
```

Comment:
"is a reflective shield" is misleading... just say something like "Ship shields are not affected by weapon hits, and reduce all subsequent direct fire weapon hits' damage by the same amount"